curry_bot.py

```diff
@@ -82,16 +92,57 @@ async def bingo(ctx):
         await ctx.send(curry_message("...done. Room created at URL: {} with password: {}\nGo do your best! Don't stumble!".format(room_url, italics(password))))
 
 
-@client.command(description="Type '!rando' to fetch the current seed link runners are playing on. Type '!rando <link>' to overwrite current seed with a new one of your choice.", brief="Get rando seed in progress")
+@client.command(description="Type '!rando' to fetch the current seed link runners are playing on.\nType '!rando <link>' to overwrite current seed with a new one of your choice.\nType '!rando presets' for a list of presets.\nType '!rando <preset>' to generate {} seeds of the given preset.".format(SEEDS_TO_GENERATE), brief="Make or get current rando seeds")
 async def rando(ctx, *args):
     global RANDO_LINKS
+    print_links = True
     if len(args) >= 1:
-        RANDO_LINKS = args
-        await ctx.send(curry_message("Rando seed links updated"))
-    await ctx.send(curry_message("Current rando seed links:"))
-    for i, link in enumerate(RANDO_LINKS):
-        await ctx.send(curry_message("Seed {}: {}".format(i+1, link)))
-
+        # Match on the start to make this command respond to 'presets' as well
+        if args[0].startswith('preset'):
+            print_links = False
+            for preset, description in RANDO_PRESETS.items():
+                await ctx.send(curry_message("Preset: {}, Description: {}".format(preset, description)))
+        else:
+            updated = False
+            preset = get_matching_preset(args[0])
+            if preset:
+                await ctx.send(curry_message("Generating seeds..."))
+                seed_base = time.time() * 1000
+                seed_floor = math.floor(seed_base)
+                # Use the nanosecond portion of the time as a pseudo-random offset, plus a constant in case that happens to be 0
+                offset = math.floor((seed_base - seed_floor) * 10000) + 500
```

Comment:
Any reason not to just use random.randrange() to generate the seed?

Reply:
This was intentional, as I wanted the links generated by CurryBot to resemble those created by the randomizer. It gives me a quick way to determine when the seed was created. It does also guarantee that two links generated by CurryBot are never the same, as unlikely as that might be.

That said, these benefits debatable, so this could be replaced by `random.randrange()`.